some fixes and updates in user/tuto documentation (#57)
iterorganization/SimDB: docs/tutorial.md

```bash
simdb remote --new iter https://simdb.iter.org/scenarios/api
simdb remote --set-default iter
```

You can test that the remote server is valid and also list the simulations available on it:

```bash
simdb remote <REMOTE_ID> test 
simdb remote <REMOTE_ID> list
```

You can now check that your simulation is valid for a given remote server, as different servers may have different 
rules and required fields:

```bash
simdb simulation validate <REMOTE_ID> <SIM_ID>
```

Typical validation issues are:
- one of the data sources (input or output) being absent or not verifying the checksum (i.e. something changed since the ingestion);
- failing to comply with the list of mandatory metadata for the targeted remote.

It it possible to know which validation schema applies on a given remote:

```bash
simdb remote <REMOTE_ID> schema -d 10
``` 

If the `validate` command results in a `validation successful` message, then you can push your simulation:

```bash
simdb simulation push <REMOTE_ID> <SIM_ID>
```

## Authentication

Whenever you run a remote command you will notice that you have to authenticate against the remote server. This can be

> avoided by creating an authentication token using for servers that allows such method (not applicable to simdb.iter.org

Reviewer: ```suggestion
avoided by creating an authentication token for servers that allow such a method (not applicable for simdb.iter.org
```
